Chapters/2-5-Chapter12.md

```diff
@@ -0,0 +1,175 @@
+## Chapter 12
+
+"Are you sure they're landing here?" asked Kipnu. "I think we came in from over there."
+
+"I'm sure of it. Nanta decides which way to turn depending on the currents. She's not too fond of her job, but she keeps the boat running smoothly."
+
+"Sounds like you know her well."
+
+"I voluntarily traveled to Ulukoma... Tends to raise a few eyebrows."
+
+The three had taken a kelping canoe and paddled out to the space where the ships came in, waiting for their moment. Kipnu looked out at the dark sea and the hint of red and infrared that lie above it. There was far more he could see now that his eyes had adjusted to the dark. He wondered at Casra's stoic nature, how thi held the same expression no matter the hardship: calm, intelligent, and resolute. However this plan turned out, he was certain that a better one didn't exist.
+
+He glanced again at the tank of fresh water he had been tasked to carry and his eyes wandered to Casra's satchel of firestarters. He wondered if thi could really start a resistance with something so small, but then he remembered that life under Shinna was a struggle and that gestures of kindness were rare. Perhaps another would take his place after such a warm welcome.
+
+"There, a bit to the North," Linyu said. They strained their eyes to see two ships on the horizon nearing the end of the sea rails.
+
+"Two ships?" asked Casra, incredulously. "And they're so low in the water."
+
+But Kipnu spared no time and began to paddle to intercept them.
+
+They drew closer, and the front ship began its turning maneuver. A huge crowd of people was thrown off the side all at once without bindings on their arms, and they flailed about in the water struggling to float in the disturbance.
```

Comment:
So I guess that's how people are generally dropped off? Ok, I guess that's why I got confused about the first chapter of this part. It's a good illustration of how little they care about the exhiles, but it feels almost comical.